src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.SetFileInformationByHandle.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal enum FILE_INFO_BY_HANDLE_CLASS : uint
```

Comment:
The name of the file does not quite make sense because of it does not contain SetFileInformationByHandle. Rename it?